src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -109,7 +117,9 @@ export const WorkspaceForm = ({
   const [icon, setIcon] = useState(defaultValues?.icon);
   const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
 
-  const [selectedFeatureIds, setSelectedFeatureIds] = useState(defaultValues?.features || []);
+  const [selectedFeatureIds, setSelectedFeatureIds] = useState(
+    defaultValues?.features || DEFAULT_CHECKED_FEATURES_IDS
+  );
```

Comment:
How about we append the default checked features into selectedFeatureIds, in such case we do not need to have the complicate check on if we should set the default checked item disabled.